packages/orchestrator-ui-components/src/components/WFOBadges/WFOFailedTasksBadge/WFOFailedTasksBadge.tsx

```diff
@@ -32,32 +34,59 @@ const getTaskCountsSummary = (
 };
 
 export const WFOFailedTasksBadge = () => {
+    const t = useTranslations('common');
     const { theme } = useOrchestratorTheme();
     const { data: processStatusCounts } = useProcessStatusCountsQuery();
     const taskCountsSummary = getTaskCountsSummary(processStatusCounts);
 
-    return (
-        <EuiToolTip
-            position="bottom"
-            content={
-                <>
-                    <div>Failed: {taskCountsSummary.failed}</div>
-                    <div>
-                        Inconsistent data: {taskCountsSummary.inconsistentData}
-                    </div>
-                    <div>
-                        API unavailable: {taskCountsSummary.apiUnavailable}
-                    </div>
-                </>
-            }
-        >
-            <WFOHeaderBadge
-                color={theme.colors.emptyShade}
-                textColor={theme.colors.shadow}
-                iconType={() => <WFOXCircleFill color={theme.colors.danger} />}
+    if (taskCountsSummary.total != 0) {
+        return (
+            <EuiToolTip
+                position="bottom"
+                content={
+                    <>
+                        <div>Failed: {taskCountsSummary.failed}</div>
```

Comment:
Translation?

Reply:
I thought we can skip this, as these are internal states which the user will also see in the table columns of the task.